test/ruby/test_zjit.rb

```diff
@@ -791,7 +810,7 @@ def assert_compiles(expected, test_script, insns: [], **opts)
       result = {
         ret_val:,
         #{ unless insns.empty?
-          'insns: RubyVM::InstructionSequence.of(_test_proc).enum_for(:each_child).map(&:to_a)'
+          'insns: RubyVM::InstructionSequence.of(method(:test)).to_a'
```

Comment:
The comment above is no longer correct.

@k0kubun do we rely, implicitly or explicitly, on this checking that *every* function is compiled? I would guess that this was introduced near your direct-call-JIT patch

Reply:
It's more like "assert that all compilation attempts are successful at a certain call threshold". I think it's a possible reading of the current comment, but maybe it's not very clear?